controllers/apply/standard-proposal/fields.js

```diff
@@ -187,21 +189,21 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
 
         function schema() {
             const isEnglandSelected = Joi.array().items(
-                Joi.string().only('england').required(),
```

Comment:
`.only` was previously an alias of `.valid` but that's changed in the latest version. Using `.valid` directly here.